R/zzz.R

```diff
@@ -193,6 +193,13 @@ settings <- NULL
     exclude = rex::rex("#", any_spaces, "nolint"),
     exclude_start = rex::rex("#", any_spaces, "nolint start"),
     exclude_end = rex::rex("#", any_spaces, "nolint end"),
+    exclude_linter = rex::rex(start, any_spaces, ":", any_spaces,
+                              capture(
+                                name = "linters",
+                                zero_or_more(one_or_more(none_of(",.")), any_spaces, ",", any_spaces),
```

Comment:
note that this means downstream custom linters can't have `.` in their name

Reply:
I have this regex lying around btw for a valid R object symbol:

```
[a-zA-Z][a-zA-Z0-9._]*|[.]|[.][a-zA-Z._][a-zA-Z0-9._]*
```

Reply:
That complicates the `: ... .` syntax.
I'll open up a discussion whether we should still change the default here while it's hot.

I think using `(` and `)` as opener and closer respectively should turn out okay, but I'd like to make this change in a new PR, correcting all our internal `# nolints` for this as well.

WDYT?